ocs_ci/ocs/resources/pod.py

```diff
@@ -2174,17 +2173,10 @@ def run_osd_removal_job(osd_ids=None):
 
     """
     osd_ids_str = ",".join(map(str, osd_ids))
-    ocp_version = get_ocp_version()
-    ocs_version = config.ENV_DATA["ocs_version"]
-
-    if Version.coerce(ocs_version) >= Version.coerce(
-        "4.10"
-    ) and not check_safe_to_destroy_status(osd_ids_str):
-        cmd = f"process ocs-osd-removal -p FORCE_OSD_REMOVAL=true -p FAILED_OSD_IDS={osd_ids_str} -o yaml"
-    elif Version.coerce(ocp_version) >= Version.coerce("4.6"):
-        cmd = f"process ocs-osd-removal -p FAILED_OSD_IDS={osd_ids_str} -o yaml"
+    if check_safe_to_destroy_status(osd_ids_str):
```

Comment:
The function gets only one osd id, so it will fail if we pass multiple osd ids. 
Maybe we can change it to this code:
`if all([check_safe_to_destroy_status(osd_id) for osd_id in osd_ids_str.split(',')]):`

Reply:
done